packages/flutter_tools/lib/src/ios/code_signing.dart

```diff
@@ -20,7 +20,7 @@ const String noCertificatesInstruction = '''
 ════════════════════════════════════════════════════════════════════════════════
 No valid code signing certificates were found
 You can connect to your Apple Developer account by signing in with your Apple ID
-in Xcode and create an iOS Development Certificate as well as a Provisioning 
+in Xcode and create an iOS Development Certificate as well as a Provisioning\u0020
```

Comment:
why?

Reply:
Because [this line](https://github.com/flutter/flutter/blob/a66ea0a628ef3bb31ba0b6c1cdf6ad657f0448a6/packages/flutter_tools/test/ios/mac_test.dart#L273) has an embedded space in the test, but I didn't want to just remove it, because it appeared that it was just mocking something that the real iOS tools output.

So I converted it to an escape sequence so that the whitespace presubmit check I have in the other PR won't fail because there's a space here.  I'd really prefer a [linter rule for EOL whitespace](https://github.com/dart-lang/linter/issues/1088), but it doesn't exist yet.